app/assets/v2/js/grants/_new.js

```diff
@@ -277,6 +277,17 @@ Vue.mixin({
           _alert(err.message, 'danger');
         }
       });
+    },
+    handleTwitterUsername(event) {
+      const inputField = event.target;
+      const matchResult = inputField.value.match(/https:\/\/twitter.com\/(\w{4,15})/);
+      let extracted;
+
+      if (matchResult) {
+        extracted = `@${matchResult[1]}`;
+      }
+
+      this.$set(this.form, inputField.id, extracted);
```

Comment:
I don't think this covers the case where a user inputs a valid twitter handle now. If they input `@gitcoin` I think it wold be set as `undefined` with this implementation.

```suggestion
      const extracted = matchResult ? `@${matchResult[1]` : inputField.value;

      this.$set(this.form, inputField.id, extracted);
```